daft/datatype.py

```diff
@@ -32,6 +33,43 @@
         _RAY_DATA_EXTENSIONS_AVAILABLE = False
 
 
+class ImageMode(str, Enum):
+    """
+    Supported image modes for Daft's image type.
+    """
+
+    B = "1"
+    L = "L"
+    P = "P"
+    LA = "LA"
+    RGB = "RGB"
+    RGBA = "RGBA"
+    CMYK = "CMYK"
+    YCbCr = "YCbCr"
+    LAB = "LAB"
+    HSV = "HSV"
+    I = "I"
+    F = "F"
```

Comment:
These are PIL-language for int32 and float32, I'm wondering if we should remove these?